lib/block_view_stake.go

```diff
@@ -2021,22 +2031,24 @@ func (bav *UtxoView) SanityCheckStakeTxn(
 		return errors.New("SanityCheckStakeTxn: StakeAmountNanos increase does not match")
 	}
 
-	// Validate GlobalStakeAmountNanos increase.
-	if utxoOp.PrevGlobalStakeAmountNanos == nil {
-		return errors.New("SanityCheckStakeTxn: nil PrevGlobalStakeAmountNanos provided")
-	}
-	currentGlobalStakeAmountNanos, err := bav.GetGlobalStakeAmountNanos()
-	if err != nil {
-		return errors.Wrapf(err, "SanityCheckStakeTxn: error retrieving CurrentGlobalStakeAmountNanos: ")
-	}
-	globalStakeAmountNanosIncrease, err := SafeUint256().Sub(
-		currentGlobalStakeAmountNanos, utxoOp.PrevGlobalStakeAmountNanos,
-	)
-	if err != nil {
-		return errors.Wrapf(err, "SanityCheckStakeTxn: error calculating GlobalStakeAmountNanos increase: ")
-	}
-	if !globalStakeAmountNanosIncrease.Eq(amountNanos) {
-		return errors.New("SanityCheckStakeTxn: GlobalStakeAmountNanos increase does not match")
+	// Validate GlobalActiveStakeAmountNanos increase if validator is active.
```

Comment:
we could also add a check that `PrevGlobalActiveStakeAmountNanos` is nil if the validator is jailed - this is a good check that ensures we don't modify the value unintentionally.